superset/advanced_data_type/plugins/tcp_sequence.py

```diff
@@ -0,0 +1,37 @@
+import re
+from superset.advanced_data_type.plugins import translate_filter_func
+from superset.advanced_data_type.plugins.operator_sets import EQUAL_NULLABLE_OPERATOR_SET
+from superset.advanced_data_type.types import AdvancedDataType, AdvancedDataTypeRequest, AdvancedDataTypeResponse
+
+def tcp_sequence_func(req: AdvancedDataTypeRequest) -> AdvancedDataTypeResponse:
+    """
+    Convert a passed in AdvancedDataTypeRequest to an AdvancedDataTypeResponse
+    """
+    resp: AdvancedDataTypeResponse = {
+        "values": [],
+        "error_message": "",
+        "display_value": "",
+        "valid_filter_operators": EQUAL_NULLABLE_OPERATOR_SET,
+    }
+    if req["values"] == [""]:
+        resp["error_message"] = "TCP sequence must not be empty"
+        return resp
+    else:
+        for val in req["values"]:
+            string_value = str(val)
+            if re.search("^[a-zA-Z]*.[a-zA-Z]*?(.[a-zA-Z0-9]+)*$", string_value):
```

Comment:
Isn't TCP Sequence just a large numerical value?

Reply:
oh whoops , must've gotten messed up in the merge. Fixed now: b28384d07bf8d0ce163a3191182bf75c04ee6bfd